Content.Server/Strip/StrippableSystem.cs

```diff
@@ -354,6 +364,14 @@ bool Check()
                 NeedHand = true,
             };
 
+            if (Check() && userHands.Hands.TryGetValue(handName, out var handSlot))
+            {
+                if (handSlot.HeldEntity != null)
+                {
+                    _popupSystem.PopupEntity(Loc.GetString("strippable-component-alert-owner-insert", ("user", user), ("item", handSlot.HeldEntity)), component.Owner, Filter.Entities(component.Owner));
+                }
+            }
```

Comment:
do you need to do `Check` again for all of these? i dont really see why its necessary

Reply:
Yes or else the message plays regardless of what happens. For example, if the check for a slot not existing fails, that message will play plus the message that someone's trying to strip something off of you. With the check, it only plays if all the checks pass.